server/server/data_loader/dictionaries.py

```diff
@@ -12,10 +12,13 @@ def load_dictionaries(db, dictionaries_dir):
 
     for dictionary in tqdm(dictionaries):
         with dictionary.open('r', encoding='utf-8') as f:
-            from_lang, to_lang = dictionary.stem.split('2')
-            docs.append({
-                'from': from_lang,
-                'to': to_lang,
-                'dictionary': json.load(f)
-            })
+            try:
+                from_lang, to_lang = dictionary.stem.split('2')
+                docs.append({
+                    'from': from_lang,
+                    'to': to_lang,
+                    'dictionary': json.load(f)
+                })
+            except ValueError:
+                pass
```

Comment:
I thought this was temporary?

Reply:
Exactly. Till the way we are supposed to hold dictionaries will be clear so it has to stay here for now

Reply:
Maybe at least print the `Exception`?

Reply:
We could. A will add something